doc/source/dev/contributor/debugging_linalg_issues.rst

```diff
@@ -477,4 +477,63 @@ file to automate this and avoid the manual paths:
    machine, it's easy to accidentally pick up the wrong one during the build
    (remember: ``-lopenblas`` only says "link against *some*
    ``libopenblas.so``). If you're not sure, use ``ldd`` on the test executable
-   you built to inspect which shared library it's linked again.
+   you built to inspect which shared library it's linked against.
+
+
+Debugging linalg issues with ``gdb``
+------------------------------------
+
+When debugging linalg issues, it is sometimes useful to step through both Python
+and C code. You can use ``pdb`` for the former, and ``gdb`` for the latter.
+
+First, prepare a small python reproducer, with a breakpoint. For example::
+
+    $ cat chol.py
+    import numpy as np
+    from scipy import linalg
+    n = 40
+    rng=np.random.refault_rng(1234)
```

Comment:
two typos here: need spaces around `=`, and need to fix `refault`->`default`

Reply:
:facepalm:  trying to use the web editor from the phone was indeed a bad idea.